src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball newBall = new Ball();
+        newBall.setColor(colorSupplier.getRandomColor());
+        newBall.setNumber(new Random().nextInt(101));
```

Comment:
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();

    public Ball getRandomBall() {
        Ball newBall = new Ball();
        newBall.setColor(colorSupplier.getRandomColor());
        newBall.setNumber(random.nextInt(101));
```
тут можна аналогічно з рандомом поступити